### ISSUE TITLE
RoC wheat shouldn't be able to produce bread by just cooking it

### ISSUE THREAD

**Issue by SvenKayser** [reteo/Survival-Industry#126]
RoC wheat can be used to directly cook it to bread. This is pretty inconsistent with the more complex approach of baking bread otherwise.

**Comment by reteo** [comment]
Are you talking about the vanilla recipe?

**Comment by SvenKayser** [comment]
No. When you grind wheat to RoC flour, you can directly cook the flour to 1 unit of bread. Considering that other bread recipes are much more complex, this really invalidates them over proportion.
